src/DynamoCoreWpf/Views/Core/DynamoView.xaml.cs

```diff
@@ -1305,6 +1305,13 @@ private void Selection_CollectionChanged(object sender, NotifyCollectionChangedE
 
         private void Controller_RequestsCrashPrompt(object sender, CrashPromptArgs args)
         {
+            if (CrashReportTool.ShowCrashErrorReportWindow(
+                (args is CrashErrorReportArgs cerArgs) ? cerArgs : 
+                new CrashErrorReportArgs(dynamoViewModel, null)))
+            {
+                return;
+            }
+            // Backup crash reporting dialog (in case ADSK CER is not found)
```

Comment:
we need to keep the old crash reporting UI because CER may not always exist on the user machine. (sad complication because we were not allowed to bundle CER with Dynamo)